wolfssl/wolfcrypt/asn.h

```diff
@@ -799,7 +799,7 @@ extern const WOLFSSL_ObjectInfo wolfssl_object_info[];
  * Any certificate containing more than this number of subject
  * alternative names will cause an error when attempting to parse. */
 #ifndef WOLFSSL_MAX_ALT_NAMES
-#define WOLFSSL_MAX_ALT_NAMES 128
+#define WOLFSSL_MAX_ALT_NAMES 1024
```

Comment:
Didn't we decide 512? I am okay with 1024 if you feel strongly about it

Reply:
I didnt remember if we decided on 512 or 1024 at the end. I would just hate to make this change and then hit another cert in the wild with 513 names only to have to change it again. 512 and 1024 are functionally the same in terms of denial of service protection to me, so I see no reason to limit at 512.